res/database/Default/database_people.xml

```diff
@@ -1378,13 +1675,14 @@
             <last_name_original>Coster-Waldau</last_name_original>
             <details job="2" gender="1" birthday="1970-07-27" deathday="" country="DK" />
         </person>
-        <person id="DF-Celebreties-Sean_Bean" tmdb_id="" imdb_id="nm0000293" creator="8782" created_by="DerFronck">
+        <person id="d300cc4e-dcba-4bbf-b9d4-b111412718e1" tmdb_id="" imdb_id="nm0000293" creator="8782" created_by="DerFronck" comment="merged duplicate. Previous id: DF-Celebreties-Sean_Bean">
```

Comment:
just checking: references to previous ID have been updated?

Reply:
yes